Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -86,14 +86,13 @@ private void MoveClosedIssuesFromPreviousVersion (SemanticVersion currentVersion
       if (jiraClosedIssuesWithFixVersion.Count == 0)
```

Comment:
Update method name to represent correct intention.
`MoveClosedIssuesFromPreviousVersion`